gulpfile.js

```diff
@@ -0,0 +1,9 @@
+const { src, dest } = require("gulp");
+
+function rdkitCopy(cb) {
+  return src(
+    "node_modules/@rdkit/rdkit/Code/MinimalLib/dist/RDKit_minimal.*"
+  ).pipe(dest("public/"));
+}
+
+exports.rdkitCopy = rdkitCopy;
```

Comment:
this is a [gulp](https://gulpjs.com/) copying the RDKit minimal lib files (js and wasm files) and copying them in the public folder (the initial place they were in before this PR, only now these files will always mirror the version of rdkit-js you have installed in your local repository.